py-polars/polars/convert.py

```diff
@@ -219,11 +243,16 @@ def from_numpy(
     └─────┴─────┘
 
     """
-    return DataFrame._from_numpy(data, columns=columns, orient=orient)
+    return DataFrame._from_numpy(
+        data, columns=columns, orient=orient, schema_overrides=schema_overrides
+    )
 
 
 def from_arrow(
-    a: pa.Table | pa.Array | pa.ChunkedArray, rechunk: bool = True
+    a: pa.Table | pa.Array | pa.ChunkedArray,
+    columns: Sequence[str] | None = None,
```

Comment:
Since we're adding a new parameter, we should give it the right name straight away.
```suggestion
    schema: Sequence[str] | None = None,
```

Reply:
Done. I've also specced a new docstring that (hopefully ;) better explains the `schema` param. Using this function as a testbed for the; new explanation see what you think?